docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx

```diff
@@ -89,56 +89,49 @@ The Frame Processor Plugin will be exposed to JS through the `VisionCameraProxy`
 
 ![Xcode "Create Bridging Header" alert](https://docs-assets.developer.apple.com/published/7ebca7212c/2a065d1a-7e53-4907-a889-b7fa4f2206c9.png)
 
-3. Inside the newly created Bridging Header, add the following code:
```

Comment:
Do we not need that anymore?

Reply:
If you meant the point 2. above, then I think yes, saying "Yes" to "Create Bridging Header" alert should stay - it will just be empty, but if someone will decide to bridge some additional ObjC to Swift later, then they will not need to manually creating and linking that bridging header